packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -130,15 +130,29 @@ const EVENTS = {
   },
 
   // Reset password confirmation
-  'screen.settings.reset-password-confirmed': {
+  'screen.reset-password-confirmed': {
     group: GROUPS.settings,
     event: 'reset_password_confirmed_view',
   },
-  'flow.settings.reset-password-confirmed.continue': {
+  'flow.reset-password-confirmed.continue': {
     group: GROUPS.settings,
     event: 'reset_password_confirmed_continue',
   },
 
+  // Reset password with recovery key verified
+  'screen.reset-password-with-recovery-key-verified': {
```

Comment:
These will all need tests in `amplitude.js`. They're quick to add and you can grab the amplitude event name from the tests to add to the spreadsheet.